src/grid_maker.cu

```diff
@@ -551,7 +551,7 @@ namespace libmolgrid {
     template<typename Dtype>
     __global__
     void set_grid_gradients(GridMaker G, float3 grid_origin, Grid2fCUDA coords, Grid1fCUDA type_index,
```

Comment:
I'm having a lot of trouble the math here.  My assumption was we wanted to calculate second order gradients.  accumulate_grid_gradient is calculating the first derivative.  How can you get second order without using the second derivative of the density function?